test/Microsoft.NET.TestFramework/ProjectConstruction/TestProject.cs

```diff
@@ -168,6 +170,11 @@ internal void Create(TestAsset targetTestAsset, string testProjectsSourceFolder)
                 targetFrameworkVersionElement.SetValue(this.TargetFrameworkVersion);
             }
 
+            foreach (var additionalProperty in AdditionalProperties)
+            {
+                propertyGroup.Add(new XElement($"{ns}{additionalProperty.Key}", additionalProperty.Value));
```

Comment:
This should be ns + additionalProperty.Key like the others above.

You want https://msdn.microsoft.com/en-us/library/system.xml.linq.xnamespace.op_addition(v=vs.110).aspx, not string concatenation.  I suspect this is only working now because the projects under test have no namespace declared.